conjur/argument_parser/_host_parser.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+from conjur.argument_parser.parser_utils import command_description, command_epilog, formatter, title_formatter
+
+
+class HostParser:
+    def add_host_parser(self):
```

Comment:
docstring?

Reply:
Something like this class builds the Host option for the argparser

Reply:
Cool Done